porting/nimble/src/nimble_port.c

```diff
@@ -55,17 +58,79 @@ nimble_port_init(void)
 #endif
 }
 
+void
+nimble_port_deinit(void)
+{
+    ble_npl_eventq_deinit(&g_eventq_dflt);
+
+    ble_hs_deinit();
+}
+
 void
 nimble_port_run(void)
 {
     struct ble_npl_event *ev;
+    int arg;
 
     while (1) {
         ev = ble_npl_eventq_get(&g_eventq_dflt, BLE_NPL_TIME_FOREVER);
         ble_npl_event_run(ev);
+        arg = (int)ble_npl_event_get_arg(ev);
+        if (arg == NIMBLE_PORT_DEINIT_EV_ARG) {
```

Comment:
My feeling is that it would be better to compare:
```
ev == &ble_hs_ev_stop
```

here rather than using an argument to identify this event.  Or at least I think it would be better to use a less likely value than `-1`.

Reply:
That's right. I totally missed that this could be used. I will update the PR

Reply:
Done!